erp/management/commands/base_duplication.py

```diff
@@ -0,0 +1,69 @@
+from django.core.management.base import BaseCommand
+
+from erp.exceptions import MergeException
+from erp.models import Erp
+
+
+class BaseDuplicationCommand(BaseCommand):
```

Comment:
It looks like we will create duplicates :-D 
I would have named it Deduplication or RemoveDuplicate, something like this.